dmaws/utils.py

```diff
@@ -103,6 +103,7 @@ def __getitem__(self, key):
 def template_string(string, variables, templates_path=None):
     jinja_env = jinja2.Environment(
         trim_blocks=True,
+        lstrip_blocks=True,
```

Comment:
It might be an idea to update Jinja to 2.10 to bring in line with the FE apps (the AWS repo has it at 2.8 currently) to avoid running into further 'why is Jinja behaving weirdly' gotchas in future...

Reply:
Yeah definitely agree...